core/services/workflows/syncer/v2/workflow_registry_test.go

```diff
@@ -155,7 +155,12 @@ func Test_generateReconciliationEventsV2(t *testing.T) {
 		require.NoError(t, err)
 
 		require.Len(t, events, 2)
-		require.Equal(t, WorkflowRegistered, events[0].Name)
+		require.Equal(t, WorkflowDeleted, events[0].Name)
```

Comment:
@justinkaseman what if the previous one is paused, and not deleted?

Registry for CRE Smartcon will support workflows with the same name to exist on the registry with different ID, and they can be paused, unpaused etc 
@eutopian can you provide detail here please?
https://docs.google.com/document/d/134i1Y1PZ9rSv7J6hFvAvv98VaL5Vgt2lgkb_2bRyUZY/edit?tab=t.0#heading=h.5mhnzed94vio